src/routes/_components/DynamicPageBanner.html

```diff
@@ -10,8 +10,9 @@ <h1 class="dynamic-page-title" aria-label={ariaTitle}>{title}</h1>
   <button type="button"
           class="dynamic-page-go-back"
           aria-label="Go back"
-          on:click="onGoBack(event)">Back</button>
+          on:click|preventDefault="onGoBack()">Back</button>
```

Comment:
This is a cool Svelte feature that I wasn't aware of, but in the future maybe it should go in another PR? :)

Reply:
Ok. I didn't realize I was adding something new... all I wanted was to avoid having to define two methods.